apps/api/src/form/entity/form.entity.ts

```diff
@@ -7,6 +7,6 @@ export class FormEntity extends BaseEntity {
   @Column('jsonb', { nullable: false })
   payload!: FormPayload;
 
-  @Column('varchar', { nullable: false })
+  @Column('text', { nullable: false })
```

Comment:
I think we must use varchar over text since the content size can be variable. the `text` type takes up fixed space and that is not necessary.